CONTRIBUTING.md

```diff
@@ -500,10 +500,13 @@ formatting and semantic checking of code. We provide a pre-commit git hook for
 performing these checks, before a commit is created:
 
   ```bash
-  pip install flake8-mypy
   ln -s ../../tools/git-pre-commit .git/hooks/pre-commit
```

Comment:
Might be nice to provide install instructions on this page but idk

Reply:
Mostly trying to keep them in one place, and the wiki is easier to keep up to date.